label.rb

```diff
@@ -0,0 +1,14 @@
+class Label
+  attr_accessor :title, :color, :items
+
+  def initialize(title, color)
+    @title = title
+    @color = color
+    @items = []
+  end
```

Comment:
@bohaz Remember that we need to include an Id like this: 
![image](https://github.com/Diegogagan2587/catalog-of-my-things/assets/61764778/190cdcfd-4b9e-428c-a879-346ed02e2138) 
According to the diagram!
![image](https://github.com/Diegogagan2587/catalog-of-my-things/assets/61764778/f95e270f-3fbe-4739-a23c-43bec9bf537d)